deluge-labeler.py

```diff
@@ -136,87 +134,106 @@ def notmet_release(release):
         or "cbfm" in lowern
         or "eleanor" in lowern
         or "accomplishedyak" in lowern
-    ):
-        return True
-    else:
-        return False
+    )
 
 
-def limited_tracker(t_object, limited):
+def limited_tracker(t_object, limited = False):
     trackers = t_object[1].get("tracker", None)
     name = t_object[1].get("name", [None])
     label = t_object[1].get("label", [None])
 
-    if len(trackers) == 0:
+    if not trackers:
```

Comment:
```suggestion
    if len(trackers) == 0:
```
reverting, when tracker is not provided due to an "error" state, or some other edge-case like the 'state' being corrupted in memory requiring a restart of the daemon, it will return and set a string of length 0, not undefined.

This was something I initially tried, and found it was ineffective, so I used len() to accomplish the desired result.

Reply:
Empty string still is falsey. All these are falsey:
```python
In [1]: if '': print('yupp')
In [2]: if 0: print('sup')
In [3]: if []: print('sup')
In [4]: 
```